demisto_sdk/commands/validate/tools.py

```diff
@@ -32,3 +34,12 @@ def collect_all_inputs_from_inputs_section(content_item: Playbook) -> Set[str]:
     inputs: dict = content_item.data.get("inputs", {})
     inputs_keys = [input["key"].strip() for input in inputs if input["key"]]
     return set(inputs_keys)
+
+
+def check_timestamp_format(timestamp):
```

Comment:
why not in `tools.py` (not under validate), the general one
check if we have something similar that already exist